frontend/lib/ui/Table/Table.tsx

```diff
@@ -61,14 +57,6 @@ function LinkRow({ children, to }: { children: React.ReactNode[]; to: To }) {
   );
 }
 
-function Cell({
-  children,
-  number,
-  fontSizeClass,
-}: {
-  children?: React.ReactNode;
-  number?: boolean;
-  fontSizeClass: string;
-}) {
-  return <td className={`${number ? `${cls.number} ` : ""}${fontSizeClass}`}>{children}</td>;
+function Cell({ children, className }: { children?: React.ReactNode; className?: string }) {
+  return <td className={className}>{children}</td>;
```

Comment:
With this change, doesnt it make more sense to just use `<td>` instead of `<Table.Cell>` if not using defined properties. 
Cell props could also extend React.HTMLAttributes perhaps? className would be included.

My main problem with this is how do you know what classes are available?